src/crystal/system/win32/time.cr

```diff
@@ -10,6 +10,8 @@ module Crystal::System::Time
   NANOSECONDS_PER_SECOND    = 1_000_000_000
   FILETIME_TICKS_PER_SECOND = NANOSECONDS_PER_SECOND / NANOSECONDS_PER_FILETIME_TICK
 
+  # TODO: For now, this method returns the UTC offset currently in place.
+  # A proper implementation is required once Crystal's Time Zone API has been revised (see issue #5324)
```

Comment:
I'd prefer not to mention unmerged PRs as if it's a certainty they will be merged. Just say that this is not a complete implementation.